examples/DhtTemperatureAndHumiditySensor/DhtTemperatureAndHumiditySensor.ino

```diff
@@ -62,95 +74,218 @@ static const uint8_t FORCE_UPDATE_N_READS = 10;
 
 #define CHILD_ID_HUM 0
 #define CHILD_ID_TEMP 1
+#define CHILD_ID_TEMP_HI 2
+
+// Set this offset if the sensors have permanent small offsets to the real temperatures/humidity.
+// In Celsius degrees or moisture percent
+#define SENSOR_HUM_OFFSET 0       // used for temperature data and heat index computation
+#define SENSOR_TEMP_OFFSET 0      // used for humidity data
+#define SENSOR_TEMP_HI_OFFSET 0   // used for heat index data
 
+
+
+#include <MySensors.h>  
+#include <Adafruit_Sensor.h>
+#include <DHT_U.h>
+
+DHT_Unified dhtu(DHTDATAPIN, DHTTYPE);
+// See guide for details on Adafruit sensor wiring and usage:
+//   https://learn.adafruit.com/dht/overview
+
+uint32_t delayMS;
 float lastTemp;
 float lastHum;
-uint8_t nNoUpdatesTemp;
-uint8_t nNoUpdatesHum;
+uint8_t nNoUpdates = FORCE_UPDATE_N_READS; // send data on start-up 
 bool metric = true;
+float temperature;
+float humidity;
+float temperatureHI;
+
+
 
 MyMessage msgHum(CHILD_ID_HUM, V_HUM);
 MyMessage msgTemp(CHILD_ID_TEMP, V_TEMP);
-DHT dht;
+MyMessage msgTempHI(CHILD_ID_TEMP_HI, V_TEMP);
+
+
+
+float computeHeatIndex(float temperature, float percentHumidity) {
+  // Based on Adafruit DHT official library (https://github.com/adafruit/DHT-sensor-library/blob/master/DHT.cpp)
+  // Using both Rothfusz and Steadman's equations
+  // http://www.wpc.ncep.noaa.gov/html/heatindex_equation.shtml
+
+  float hi;
+
+  temperature = temperature + SENSOR_TEMP_OFFSET; //include TEMP_OFFSET in HeatIndex computation too
+  temperature = 1.8*temperature+32; //convertion to *F
+
+  hi = 0.5 * (temperature + 61.0 + ((temperature - 68.0) * 1.2) + (percentHumidity * 0.094));
+
+  if (hi > 79) {
+    hi = -42.379 +
+             2.04901523 * temperature +
+            10.14333127 * percentHumidity +
+            -0.22475541 * temperature*percentHumidity +
+            -0.00683783 * pow(temperature, 2) +
+            -0.05481717 * pow(percentHumidity, 2) +
+             0.00122874 * pow(temperature, 2) * percentHumidity +
+             0.00085282 * temperature*pow(percentHumidity, 2) +
+            -0.00000199 * pow(temperature, 2) * pow(percentHumidity, 2);
+
+    if((percentHumidity < 13) && (temperature >= 80.0) && (temperature <= 112.0))
+      hi -= ((13.0 - percentHumidity) * 0.25) * sqrt((17.0 - abs(temperature - 95.0)) * 0.05882);
+
+    else if((percentHumidity > 85.0) && (temperature >= 80.0) && (temperature <= 87.0))
+      hi += ((percentHumidity - 85.0) * 0.1) * ((87.0 - temperature) * 0.2);
+  }
+
+  hi = (hi-32)/1.8;
+  return hi; //return Heat Index, in *C
+}
+
+
+
+
 
 
 void presentation()  
 { 
   // Send the sketch version information to the gateway
-  sendSketchInfo("TemperatureAndHumidity", "1.1");
-  
+  sendSketchInfo(SN, SV);
   // Register all sensors to gw (they will be created as child devices)
-  present(CHILD_ID_HUM, S_HUM);
-  present(CHILD_ID_TEMP, S_TEMP);
-  
+  present(CHILD_ID_HUM, S_HUM, "Umidity");
```

Comment:
Typo: "Humidity"

Reply:
same answer than before: 
I'm going to correct it.
thank you